libraries/RC_Channel/RC_Channel.h

```diff
@@ -651,6 +659,10 @@ class RC_Channels {
 
     void set_aux_cached(RC_Channel::AUX_FUNC aux_fn, RC_Channel::AuxSwitchPos pos);
 #endif
+
+    bool clear_overrides_by_rc(void) const {
```

Comment:
Replace this with a call to `option_is_enabled`

Reply:
Replaced it with a call to option_is_enabled. Thanks for pointing it out.